README.md

```diff
@@ -78,23 +79,23 @@ After you're finished please remove all the comments and instructions!
 ### Key Features <a name="key-features"></a>
 
 
-- **[Navigation bar]**
-- **[About section]**
-- **[Contact Form]**
-- **[Projects Popup]**
+- **[ Home Page]**
+- **[About Page]**
+- **[Responsive Layout]**
+- 
 <!-- - **[key_feature_3]** -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-<!-- ## 🚀 Live Demo <a href="kimitawanjohi.live" name="live-demo">kimitawanjohi.live</a> -->
+## 🚀 Live Demo <a href="pulse.kimitawanjohi.live" name="live-demo">pulse.kimitawanjohi.live</a>
 
```

Comment:
- [ ] Good you add the live demo section in your readme :+1:  but I think the link to the live demo should be under the `Live Demo` header.